build.gradle.kts

```diff
@@ -10,6 +10,7 @@ dependencyCheck {
 
 configurations {
   implementation { exclude(mapOf("module" to "tomcat-jdbc")) }
+  testImplementation { exclude(group = "org.junit.vintage") }
```

Comment:
needed for gradle upgrade - vintage now seems to get upset if junit not available